Installers/Linux/RUN/postinstall.sh

```diff
@@ -88,6 +84,18 @@ case "$choice2" in
 	sudo -H -u $SUDO_USER bash -c 'mdtool setup install $DIR/Main/MonoDevelop.MonoGame.mpack'
 esac
 
+#MonoGame.Content.Builder.targets
+
+if [ ! -d /usr/lib/mono/xbuild/MonoGame ]; then
+	mkdir /usr/lib/mono/xbuild/MonoGame
+fi
+
+if [ ! -d /usr/lib/mono/xbuild/MonoGame/v3.0 ]; then
+	mkdir /usr/lib/mono/xbuild/MonoGame/v3.0
+fi
+
+sudo cp $DIR/Main/MonoGame.Content.Builder.targets tmp/usr/lib/mono/xbuild/MonoGame/v3.0/
```

Comment:
Linux assemblies are installed to:
/usr/lib/mono/xbuild/MonoGame/v3.0/Assemblies/Linux/****

> You can get the idea from https://github.com/mono/MonoGame/blob/develop/Installers/default.build#L64

'Linux is the only thing that can be compiled from Linux on that list, so no point in including anything else I guess.